tests/test_suite/start.py

```diff
@@ -41,6 +41,14 @@ def __init__(self, root_path, test_name, platform, architecture):
     if not os.path.isfile(self._binary_path):
       raise IOError("The binary file does not exists")
 
+    tob_lib_repository = os.environ["TRAILOFBITS_LIBRARIES"]
```

Comment:
<3 tests

Can you also copy this over the `start.py` in `test_suite_generator` so its auto-updated ?